### PR TITLE
N49: space_editor

### PR DESCRIPTION
**Pull request by rewin123**
_Part of #1440

### CODE REVIEW

File: content/news/049/space_editor.png
Comment (AngelOnFira) [inline content/news/049/space_editor.png:1]: If we could convert this to a PNG, that would be great! :)

File: content/news/049/index.md [hunk 1]
```diff
@@ -73,6 +73,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+   This enables quick and efficient development iteration.
+4. Save your levels and object templates in the standard Bevy scene format 
+   and spawn in game with just one line of code
+5. Many custom structures that allow to customize saved scenes 
+   equally conveniently, both via gui and directly editing the saved text file
+
+The [Github project](https://github.com/rewin123/space_editor) is free for use 
+and open to contribute for everyone, be it with suggestions, bugs or PRs.
+
+Link to project: https://github.com/rewin123/space_editor
```
Comment (AngelOnFira) [inline content/news/049/index.md:98]:
```suggestion
The [Github project][space_editor] is free for use 
and open to contribute for everyone, be it with suggestions, bugs or PRs.

[space_editor]: https://github.com/rewin123/space_editor
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

File: content/news/049/space_editor.png
Comment (rewin123) [reply content/news/049/space_editor.png:1]: I don't understand the question a little bit. Isn't the image already in png format? 😅

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for the PR (and sorry for the late review)!

**Pull request merged**